sys-apps/tomoyo-tools/tomoyo-tools-2.5.0_p20130214-r1.ebuild

```diff
@@ -35,7 +37,7 @@ src_prepare() {
 		-e 's|-lncurses|$(shell ${PKG_CONFIG} --libs ncurses)|g' \
 		usr_sbin/Makefile || die
 
-	echo "CONFIG_PROTECT=\"/usr/$(get_libdir)/tomoyo/conf\"" > "${T}/50${PN}"
+	echo "CONFIG_PROTECT=\"/usr/$(get_libdir)/tomoyo/conf\"" > "${T}/50${PN}" || die
```

Comment:
do this in `src_install`, and do the pipe-straight-into-`newenvd`-idiom

Reply:
done